### PR TITLE
Update logo on loading page

### LINKED ISSUE #75: Update loading page with new logo
This is a simple task. Update the old logo to the new logo created from Juan and ensure mobile responsiveness in the loading page. 

I tested it out in the community page to see what it looks. 

Screenshot 2024-03-26 at 5 11 56 PM

### PR DESCRIPTION
**Pull request by dafneMena**
## Description
This PR simply updates the loading page which contains the temp Roadwatch logo to the new logo. 

## Related Issue
Closes #75. 

## Type of Changes
Please select the relevant option:
- [ ] Bug fix
- [ ] New feature
- [x] Enhancement
- [ ] Documentation update
- [ ] Other (please specify)
      
Switched out the old logo to new logo. 

## Things to Update
Please add any features, buttons, or pages that might need updating in the future:
- [ ] Button Functunality 
- [ ] Code Documentation 
- [ ] Mobile Responsiveness 
- [ ] Code testing 
- [ ] Other (please specify)

I don't think the loading page is actually used anywhere yet? 

## Screenshots 
old logo 
Screenshot 2024-04-16 at 12 54 08 PM

updated logo 
Screenshot 2024-04-16 at 12 55 33 PM

## Additional Notes
N/A

### CODE REVIEW

**Review by Aligary — APPROVED** [review]
Hi Dafne, it seems that this PR also updates the 'help' button functionality. (#115) You can close that issue as well. Overall, great work!

File: client/src/components/Loading/Loading.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import styles from './Loading.module.scss';
-import logo from '../../assets/Temp_Roadwatch_logo.svg';
+import logo from '../../assets/Updated_Roadwatch_Logo.svg';
```
Comment (Aligary) [inline client/src/components/Loading/Loading.tsx:2]: I think this import should be `import logo from '../../assets/Updated_RoadWatch_Logo.svg';` with a capital 'W'. Github is yelling at you for it.

**Review by KingJordan152 — APPROVED** [review]
Great work on implementing this, Dafne. It's good that we purge that old logo out wherever it is. 

It seems like this PR is taking into account changes made in #124. I would merge that one in then this one to avoid conflicts in the future.

**Review by morenj — APPROVED** [review]
Looks good, great job. Thank you!

**Comment by KingJordan152** [comment]
@Linkaloo Please authorize.

**Pull request merged**